build.sbt

```diff
@@ -1,40 +1,50 @@
 import Dependencies._
 import sbt.url
+import ReleaseTransformations._
 
-ThisBuild / version := (version in ThisBuild).value
 ThisBuild / organization := "uk.gov.nationalarchives"
 ThisBuild / organizationName := "National Archives"
 
 scalaVersion := "2.13.3"
 
-ThisBuild / scmInfo := Some(
+scmInfo := Some(
   ScmInfo(
     url("https://github.com/nationalarchives/tdr-generated-graphql"),
     "git@github.com:nationalarchives/tdr-generated-graphql.git"
   )
 )
-ThisBuild / developers := List(
+
+developers := List(
   Developer(
     id    = "SP",
```

Comment:
Do we have a generic account we could use here?

Reply:
Yeah good point. I'll put the Jenkins account in there.